_data/venue.yml

```
@@ -69,3 +69,6 @@
 -
   name: "Katharine Crooks"
   image: kcrooks1006.jpg
+-
+  name: "Patricio Viera Perez"
+  image: PatoViera.jpg
```

Almost there. The instructions are not super clear about this but the  of the image you add in the .yml file as to be the name of image file you added in the img/venue folder.

```suggestion
  image: IMG_0056.jpg 
```